library/src/main/java/com/novoda/downloadmanager/lib/NotificationCustomiser.java

```diff
@@ -2,7 +2,9 @@
 
 import android.support.v4.app.NotificationCompat;
 
+import com.novoda.downloadmanager.Download;
+
 public interface NotificationCustomiser {
 
-    void modifyQueuedOrDownloadingNotification(NotificationCompat.Builder builder, long batchId);
+    void modifyQueuedOrDownloadingNotification(NotificationCompat.Builder builder, Download download);
```

Comment:
right :) sorry for that.

but so it means, that for the client of the lib we only expose the concept of `Download`? 
if thats the case, i couldnt agree more :+1: 

however, this is confusing for the internal of the library but this will be sorted out one day (i guess)